resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh

```diff
@@ -85,6 +85,7 @@ case "$1" in
       --cores $SPARK_EXECUTOR_CORES
       --app-id $SPARK_APPLICATION_ID
       --hostname $SPARK_EXECUTOR_POD_IP
+      --resourceProfileId $SPARK_RESOURCE_PROFILE_ID
```

Comment:
This will default to 0 if unspecified right ?

Reply:
if the --resourceProfileId parameter isn't specified then it defaults to 0, which is default profile, so things won't work properly for stage level scheduling, it will just look like another default executor.  Is that what you meant?

Reply:
Yes, thanks for clarifying !